### ISSUE TITLE
Validation Issue for "Stripe Checkout" Option

### ISSUE THREAD

**Issue by zwiesman31** [woocommerce/woocommerce-gateway-stripe#576]
#### Affected ticket(s)

1015074-zen

#### What I expected

Checkout fields need to be validated before being able to enter payment information with Stripe Checkout.

#### What happened instead

Checkout fields don't need to be validated before being able to enter payment information with the Stripe Checkout option. Only after payment details are entered does it try to validate the checkout field information.

If the information is missing, the customer needs to enter the required information in the checkout fields, then re-add their payment informaton.

Screencast: http://cld.wthms.co/yPFmoo

#### Steps to reproduce the issue

1. Setup Stripe and enable "Stripe Checkout" option
2. Add product to cart and head to checkout page
3. Try to pay for order first without filling in fields

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

**Comment by roykho** [comment]
This is a "won't fix" please refer to #446 

Here is a breakdown of pro/cons

<strong>Pros</strong>
* Better UI.

<strong>Cons</strong>
* Breaks checkout if people have custom fields.
* Redundant validation. Once pre checkout, once again after submit.
* Maintaining redundant code that is in WC core.

<strong>Options</strong>
* They can choose not to use Stripe Checkout ( use inline form instead ).
* They can add their own validation code in, hooks are available since 4.0.6.

<strong>Summary</strong>
This does not break the checkout, just annoyances. Where as when we did have validation, it broke people's checkout when they had custom fields. So I rather it not break people's checkout.
